commit 959fefe53ce32c2c0962a4b1014ab0a879c9faa4
Author: Stefan Wahren <stefan.wahren@i2se.com>
Date:   Sat Oct 13 20:19:13 2018 +0200

    staging: bcm2835-audio: Enable compile test
    
    Enable the compilation test for bcm2835-audio.
    
    Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/Kconfig b/drivers/staging/vc04_services/bcm2835-audio/Kconfig
index 9f536533c257..62c1c8ba4ad4 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/Kconfig
+++ b/drivers/staging/vc04_services/bcm2835-audio/Kconfig
@@ -1,6 +1,6 @@
 config SND_BCM2835
         tristate "BCM2835 Audio"
-        depends on ARCH_BCM2835 && SND
+        depends on (ARCH_BCM2835 || COMPILE_TEST) && SND
         select SND_PCM
         select BCM2835_VCHIQ
         help
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1702_959fefe53ce32c2c0962a4b1014ab0a879c9faa4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1702_959fefe53ce32c2c0962a4b1014ab0a879c9faa4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
